src/InvocationModel.ts

```diff
@@ -53,6 +54,7 @@ export class InvocationModel implements coreTypes.InvocationModel {
             retryContext: fromRpcRetryContext(req.retryContext),
             traceContext: fromRpcTraceContext(req.traceContext),
             triggerMetadata: fromRpcTriggerMetadata(req.triggerMetadata, this.#triggerType),
+            options: fromRpcBindings(this.#bindings),
```

Comment:
My biggest issue with this implementation is that this now requires the `options` as input to the constuctor of `InvocationContext`, making the unit testing scenario harder. This code you have in the Wiki [here](https://github.com/Azure/azure-functions-nodejs-library/wiki/Azure-Functions-Node.js-Framework-v4) would no longer work:

```TS
const testInvocationContext = new InvocationContext({
  functionName: 'testFunctionName',
  invocationId: 'testInvocationId',
  logHandler: (_level, ...args) => console.log(...args)
});
```

Is there a way around this? Could we have the options object initialized inside the constructor, or set here after the `context = new InvocationContext({})` line?

Reply:
I assume this feedback applies to all properties on `InvocationContextInit`? I've updated so in testing scenarios you can just do `new InvocationContext()`

Reply:
That's great! Yeah, I was gonna suggest that too but in a separate PR 😛